shared/assets/Icons/IconCalendar/IconCalendar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "../icon.css";
+interface Props {
+  isContrastIconCalendar?: boolean;
+}
+
+export const IconCalendar = ({ isContrastIconCalendar }: Props) => {
+  const mode = isContrastIconCalendar ? "dark" : "light";
```

Comment:
```suggestion
export const IconCalendar = ({ isContrast }: { isContrast?: boolean }) => {
  const mode = isContrast ? "dark" : "light";
```

`...IconCalendar` здесь лишнее; отдельная модель для одного prop'а - тоже